java-checks-test-sources/src/main/java/checks/StaticMembersAccessCheck.java

```diff
@@ -46,26 +46,26 @@ public void noncompliant() {
     third.counter ++; // Noncompliant
     first.d.counter++; // Noncompliant
     first.c.d.counter++; // Noncompliant [[sc=5;ec=22;quickfixes=qf3]]
-    // fix@qf3 {{Replace "first.c.d" by "StaticMembersAccessCheckD"}}
+    // fix@qf3 {{Replace the expression by "StaticMembersAccessCheckD"}}
```

Comment:
In fact, looking at where the main issue is located, I feel it might not be the best phrasing (sorry I missed that in my first review).
What about using
`Replace by a static reference to "StaticMembersAccessCheckD"` 
instead of 
`Replace the expression by "StaticMembersAccessCheckD"`

Because we highlight the whole expression, it sounds strange.

Now, if you want to try it first and see how it goes once dogfooded, it's OK for me as well.